### PR TITLE
[v7r1] JSON serialization: cast the result when it is needed

### PR DESCRIPTION
**Pull request by zmathe**
BEGINRELEASENOTES

Please follow the template:
*RSS
FIX: JSON: convert the dictionary keys to number, when it is required.

ENDRELEASENOTES

### CODE REVIEW

File: ResourceStatusSystem/Command/TransferCommand.py [hunk 1]
```diff
@@ -7,6 +7,8 @@
 from datetime import datetime, timedelta
 
 from DIRAC import S_OK, S_ERROR
+from DIRAC.Core.DISET.RPCClient import RPCClient
```
Comment (fstagni) [inline ResourceStatusSystem/Command/TransferCommand.py]: Why this import?

Comment (zmathe) [reply ResourceStatusSystem/Command/TransferCommand.py]: This is coming from a bad merge.

**Comment by fstagni** [comment]
Please check what's wrong with the test, and then we can approve

**Comment by zmathe** [comment]
@fstagni I do not know what is wrong. The error message:
```
 Build finished. The HTML pages are in build/html.
********************************************************************************
Warnings When Creating Doc:
********************************************************************************
/home/runner/work/DIRAC/DIRAC/FrameworkSystem/Service/MonitoringHandler.py:docstring of DIRAC.FrameworkSystem.Service.MonitoringHandler.MonitoringHandler.export_commitMarks:6: WARNING: Unexpected indentation.
/home/runner/work/DIRAC/DIRAC/FrameworkSystem/Service/MonitoringHandler.py:docstring of DIRAC.FrameworkSystem.Service.MonitoringHandler.MonitoringHandler.export_commitMarks:11: WARNING: Definition list ends without a blank line; unexpected unindent.
/home/runner/work/DIRAC/DIRAC/FrameworkSystem/Service/MonitoringHandler.py:docstring of DIRAC.FrameworkSystem.Service.MonitoringHandler.MonitoringHandler.export_commitMarks:13: WARNING: Unexpected indentation.
********************************************************************************
##[error]Process completed with exit code 1.
```
tests/checkDocs.sh is failing. I do not know how to fix it.

**Comment by andresailer** [comment]
I will have a look

**Comment by andresailer** [comment]
Made the example an explicit code block with `::` and empty lines around it. The indentation change for the single numbers are just for "niceness"

**Comment by zmathe** [comment]
@andresailer Thanks for having a look. How did you find out the problem? I searched for error but I saw no error. I wanted to know, if this will happen again where to look.

**Comment by andresailer** [comment]
@zmathe There are no warnings allowed in the documentation build. And the warning clearly points to which function docstring is causing the warnings.

These warnings usually point towards things that go wrong in the "typesetting" of the documentation.

The problem here is the indentation was kind of changing randomly between from one line to the next, so this either means there has to be a newline, or an explicit code block has to be created, or indentation has to be aligned from line to line, depending on what one wants to do

**Pull request merged**